robot-core/src/main/java/org/obolibrary/robot/RelatedObjectsHelper.java

```diff
@@ -947,12 +949,13 @@ private static OWLAnnotation getLiteralAnnotation(
       OWLAnnotationProperty annotationProperty,
       String value)
       throws Exception {
-    // ioHelper.addPrefix("xsd", "http://www.w3.org/2001/XMLSchema#");
     OWLAnnotationValue annotationValue;
-    if (value.contains("^^")) {
+    Matcher datatypeMatcher = Pattern.compile("'(.*)'\\^\\^([a-z]*)").matcher(value);
+    Matcher langMatcher = Pattern.compile("'(.*)'@([a-z]*)").matcher(value);
```

Comment:
I don't think `[a-z]*` is general enough to handle `'foo'^^<http://foo>`, `'foo'@en-US`. It should probably be `.*`.

As usual, I'd feel better if there were some tests.